Code/Framework/AzFramework/AzFramework/Entity/EntityDebugDisplayBus.h

```diff
@@ -62,6 +63,7 @@ namespace AzFramework
         virtual void DrawLine(const AZ::Vector3& p1, const AZ::Vector3& p2, const AZ::Vector4& col1, const AZ::Vector4& col2) { (void)p1; (void)p2; (void)col1; (void)col2; }
         virtual void DrawLines(const AZStd::vector<AZ::Vector3>& lines, const AZ::Color& color) { (void)lines; (void)color; }
         virtual void DrawPolyLine(const AZ::Vector3* pnts, int numPoints, bool cycled = true) { (void)pnts; (void)numPoints; (void)cycled; }
+        virtual void DrawPolyLine(AZStd::span<const AZ::Vector3> points, bool cycled = true) { (void)points; (void)cycled; }
```

Comment:
Nit: Probably should be `const AZStd::span<const AZ::Vector3>`

Reply:
Does it make a difference unless it's a reference? Unless you meant `const AZStd::span<const AZ::Vector3>&`.

Reply:
you pass the span by copy since it just a start and a size. I don't think you apply const to the span?

Reply:
Apparently I need more coffee. You're 100% right, it doesn't _really_ make a difference without the reference. 

That said, I have a slight leaning towards const reference vs pass-by-value, just to avoid some unnecessary trivial construction/destruction, but not enough to even offer it up as a suggestion here.